From 171153d421e67b3679831824f5a08d1312485661 Mon Sep 17 00:00:00 2001
Message-Id: <patch-5.5-171153d421e-20211004T015432Z-avarab@gmail.com>
In-Reply-To: <cover-0.5-00000000000-20211004T015432Z-avarab@gmail.com>
References: <cover-0.5-00000000000-20211004T015432Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Fri, 6 Aug 2021 13:23:47 +0200
Subject: [PATCH 5/5] advice.[ch]: sort enum fields in alphabetical order
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

These fields have slightly drifted out of order, let's re-order them.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 advice.c | 2 +-
 advice.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/advice.c b/advice.c
index f7cedac7549..e0ee8ca4f2f 100644
--- a/advice.c
+++ b/advice.c
@@ -51,9 +51,9 @@ static struct {
 	[ADVICE_PUSH_ALREADY_EXISTS]			= { "advice.pushAlreadyExists" },
 	[ADVICE_PUSH_FETCH_FIRST]			= { "advice.pushFetchFirst" },
 	[ADVICE_PUSH_NEEDS_FORCE]			= { "advice.pushNeedsForce" },
-	[ADVICE_PUSH_REF_NEEDS_UPDATE]			= { "advice.pushRefNeedsUpdate" },
 	[ADVICE_PUSH_NON_FF_CURRENT]			= { "advice.pushNonFFCurrent" },
 	[ADVICE_PUSH_NON_FF_MATCHING]			= { "advice.pushNonFFMatching" },
+	[ADVICE_PUSH_REF_NEEDS_UPDATE]			= { "advice.pushRefNeedsUpdate" },
 	[ADVICE_PUSH_UNQUALIFIED_REF_NAME]		= { "advice.pushUnqualifiedRefName" },
 	[ADVICE_PUSH_UPDATE_REJECTED]			= { "advice.pushUpdateRejected" },
 	[ADVICE_RESET_QUIET_WARNING]			= { "advice.resetQuiet" },
diff --git a/advice.h b/advice.h
index ab25c6ba9a0..a6aca9684c7 100644
--- a/advice.h
+++ b/advice.h
@@ -31,9 +31,9 @@ enum advice_type {
 	ADVICE_PUSH_NEEDS_FORCE,
 	ADVICE_PUSH_NON_FF_CURRENT,
 	ADVICE_PUSH_NON_FF_MATCHING,
+	ADVICE_PUSH_REF_NEEDS_UPDATE,
 	ADVICE_PUSH_UNQUALIFIED_REF_NAME,
 	ADVICE_PUSH_UPDATE_REJECTED,
-	ADVICE_PUSH_REF_NEEDS_UPDATE,
 	ADVICE_RESET_QUIET_WARNING,
 	ADVICE_RESOLVE_CONFLICT,
 	ADVICE_RM_HINTS,
-- 
2.33.0.1404.g7bcfc82b295

